.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ commands:
 
                 command: |
                   mv .riot .ddriot
-                  echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9 | tr ',' '\n' | circleci tests split | xargs ./scripts/ddtest riot -v run --exitfirst --pass-env -s '<< parameters.pattern >>'
+                  riot run -s avail_pythons | head | head -1 | tr ',' '\n' | circleci tests split | xargs -I PY ./scripts/ddtest riot -v run --python=PY --exitfirst --pass-env -s '<< parameters.pattern >>'
```

Comment:
```suggestion
                  riot -P run s avail_pythons | head -n 1 | tr ',' '\n' | circleci tests split | xargs -I PY ./scripts/ddtest riot -v run --python=PY --exitfirst --pass-env -s '<< parameters.pattern >>'
```

Reply:
yeah i did try the riot pipe flag but riot still fails with an exit code of 1 😞

Reply:
```
root@10acc32ebacb:~/project# riot -P run -s avail_pythons | head -n1 
2.7,3.5,3.6,3.7,3.8,3.9,3.10
root@10acc32ebacb:~/project#  echo "${PIPESTATUS[0]} ${PIPESTATUS[1]}"
1 0
```

Reply:
Weird 😞

Reply:
```
python -c "from riotfile import select_pys; print('\n'.join(select_pys()))"
```

?

Reply:
yeah at this point we might as well just do the `-c`

Reply:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "riotfile.py", line 2, in <module>
    from typing import List
ImportError: No module named typing

Exited with code exit status 1
```

keeps getting better! (we're stuck to using 3.6 in the machine-executor)